docs/README_ko.md

```diff
@@ -0,0 +1,79 @@
+<h1 align="">Indonesian Area API (<i>API Wilayah Indonesia</i>)</h1>
```

Comment:
The title also need to be translated. Just remove the "(<i>API Wilayah Indonesia</i>)"

Reply:
Idk if I understood this correctly. Let me know after checking!